src/Tribe/Query.php

```diff
@@ -149,16 +149,29 @@ public static function parse_query( $query ) {
 			}
 
 			// never allow 404 on month view
-			if ( $query->is_main_query() && $query->get( 'eventDisplay' ) == 'month' && ! $query->is_tax && ! $query->tribe_is_event_category ) {
+			if (
+				$query->is_main_query()
+				&& $query->get( 'eventDisplay' ) == 'month'
```

Comment:
Yoda

Reply:
Right! Let me update that one as I just updated the format not those lines.